docs/usermanual.md

```diff
@@ -98,17 +98,130 @@ hello
 
 then the resulting attribute sent to ContextBroker:
 
-{
-   ...
-   "attrHex": {
-     "value": "68656c6c6f"
-     "type": "<the one used at provisiong time for attrHex attribute>"
-   }
-}
+{ ... "attrHex": { "value": "68656c6c6f" "type": "<the one used at provisiong time for attrHex attribute>" } }
+
+Note that every group of 2 character (I.E, the first group, `68`) corresponds to a single ASCII character or byte
+received in the payload (in this case, the value `0x68` corresponds to `h` in ASCII). You can use one of the multiple
+tools available online like [this one](https://string-functions.com/string-hex.aspx)
+
+##### NGSI-v2 and NGSI-LD Measure reporting
+
+It is possible report as a measure a NGSI-v2 or NGSI-LD payload when related device/group is configured with payloadType
+`ngsiv2` or `ngsild`. In these cases payload is ingested as measure where entity attributes are measure attributes and
+`id` and `type` are ignored, since `id` and `type` from device/group configuration provisioned are used, as well as
+`actionType`.
+
+Examples of these `ngsi-v2` payloads are:
+
+```
+ {
+     "actionType": "append",
+     "entities": [
+        {
+          "id": "MyEntityId1",
+          "type": "MyEntityType1",
+          "attr1": { type: "Text", "value": "MyAttr1Value"},
+          ...
+        },
+        ...
+    ]
+ }
+```
+
+and (NGSI-v2 plain entities array format):
+
+```
+ [
+    {
+          "id": "MyEntityId1",
+          "type": "MyEntityType1",
+          "attr1": { "type": "Text", "value": "MyAttr1Value"},
+          ...
+    },
+    ...
+ ]
+```
+
+Examples of these `ngsi-ld` payloads are:
+
+```
+ {
```

Comment:
Should be a proper JSON (i.e. using "..." in key names, using " instead of ', etc.).

Should `append` (in lowercase) be used here? @mapedraza NGSI-LD uses `APPEND` or `append`?

Reply:
NGSI-LD defines the action type in the URL. This means that an NGSI-LD API exposes the following routes:

![32efaadf-9349-43c9-945b-b6bff0668fa8](https://github.com/telefonicaid/iotagent-json/assets/40356341/1a1ffafa-6d1d-4fc3-8820-fcd8f87331b7)

As example of batch update, you can find the next request:

```
POST /ngsi-ld/v1/entityOperations/update HTTP/1.1
Host: localhost:9090
Content-Type: application/json
Link: <https://uri.etsi.org/ngsi-ld/primer/store-context.jsonld>; rel="http://www.w3.org/ns/jsonld#context"; type="application/ld+json"
[
  {
    "id": "urn:ngsi-ld:Store:004",
    "type": "Store",
    "address": {
      "type": "Property",
      "value": {
        "streetAddress": "Tiger Street 7",
        "addressRegion": "Metropolis",
        "addressLocality": "Cat City",
        "postalCode": "42420"
      },
      "observedAt": "2019-09-12T17:11:07Z"
    }
  },
  {
    "id": "urn:ngsi-ld:Shelf:234",
    "type": "Shelf",
    "maxCapacity": {
      "type": "Property",
      "value": 85
    }
  }
]
```

Note the `@context` element is present in the `Link` header instead of the payload

Reply:
Taking into account the above feedback, the example in L148-L186 seems doesn't having sense (as actionType is not allowed) so I suggest removing it and leave only the examples in L191-L225.

What do you think @mapedraza ?

Reply:
Undone in https://github.com/telefonicaid/iotagent-json/pull/779/commits/75b7a9d88187ba38f6a2f7759ec19cfee01226eb